osconfig_tests/gcp_clients/gcp_clients.go

```diff
@@ -0,0 +1,32 @@
+package gcpclients
```

Comment:
Missing copyright header? Here and elsewhere.

Reply:
done